client/app/components/Viewer.jsx

```diff
@@ -1,8 +1,8 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import MapContainer from '../containers/MapContainer';
-import PlanFormContainer from '../containers/PlanFormContainer';
+import MapContainer from './../containers/MapContainer';
```

Comment:
Isn't this a redundancy?

Reply:
Generally, yes. I was trying to debug what I thought was an import error in Webpack at one point, and I wanted to give this a try. It's just a leftover remnant of that test that can now be removed.